src/EditorFeatures/TestUtilities/Workspaces/TestWorkspace.cs

```diff
@@ -642,5 +642,35 @@ public void ChangeSolution(Solution solution)
 
             this.RaiseWorkspaceChangedEventAsync(WorkspaceChangeKind.SolutionChanged, oldSolution, newSolution);
         }
+
+        public bool TryGetDocumentWithSelectedSpan(out Document document, out TextSpan span)
+            => TryGetDocumentWithSelectedSpan(allowLinkedFile: true, out document, out span);
+
+        public bool TryGetDocumentWithSelectedSpan(
```

Comment:
Considerr merging the two overloads by moving `bool  allowLinkedFiile = fallse` parameter to  the end of the parameter list. Then only one caller needs too use that parameter (to set it to true).

Reply:
in general, we odn't put input parameter *after* output parameters.  Feels skeezy to me :D